index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>My portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <nav class="flex-toolbar">
+      <div class="logo-name"><p>Jesus Molina</p></div>
+      <div class="menu-flex">
+        <img id="menu" src="resources/Menu-button/Enabled.svg" alt="A menu button" />
+      </div>
+    </nav>
+
+    <section class="headline-section">
+      <div id="primary-text">
+        <p>Hey There.<br> I'm Jesus</p>
+      </div>
+      <div id="supporting-text">
+        <p>I'm a Software Developer</p>
+      </div>
+      <div id="description">I can help you build a product , feature or website Look through some of 
+        my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </div>
+      <div class="social-media">
+        <img src="resources/Menu-button/cat.svg"/>
+        <img src="resources/Menu-button/linkedin.svg"/>
+        <img src="resources/Menu-button/hand.svg"/>
+        <img src="resources/Menu-button/twitter.svg"/>
+        <img src="resources/Menu-button/m.svg"/>
+      </div>
```

Comment:
- [ ] This is a list of social links so an `<ul>` tag to group them all and a `<li>` tag to wrap each of them would be better appropriate for semantic reasons. Apart from being semantically correct, the `<a>` tag will let you connect your social networks with the icons, using `href`. Kindly change the `<div>` and add the other HTML elements for the reasons stated.

Reply:
Fixed